README.md

```diff
@@ -676,6 +677,22 @@ inspector.show_correlation()
   </tbody>
 </table>
 
+## Scope
```

Comment:
Can't leave a comment but something renders funky right above this table.

Reply:
Yeah, it shows up correctly in the notebook and on the documentation site but not on GitHub. We could solve that problem by switching from Pandas `Styler` objects to something like Seaborn heatmaps, but the `Styler` objects have some advantages such as being able to copy text out of them and to control how they display through Pandas display options.

Reply:
I'm at least OK for now with saying that this library is primarily targeting notebook environments, but we do have #3 to make the relevant methods more portable.

Reply:
Actually I think the best reason to use a `Styler` object rather than a plot is that you can use `.data` to access the underlying DataFrame for further manipulation.